src/components/new-request/search.vue

```diff
@@ -96,10 +96,24 @@
         </v-tooltip>
       </v-col>
 
-      <!-- Business Lookup -->
+      <!-- Business Lookup/Fetch -->
       <v-col v-if="showBusinessLookup" cols="12" md="6">
-        <BusinessLookup v-if="!business" @business="business=$event"/>
-        <v-text-field v-else clearable disabled filled hide-details :value="business.name" />
+        <template v-if="!business">
+          <BusinessLookup v-if="getIsAuthenticated" @business="business=$event"/>
+          <BusinessFetch v-else @business="business=$event"/>
+        </template>
+        <div v-else class="d-flex justify-space-between align-center">
+          <v-text-field
+            append-outer-icon="mdi-close"
+            disabled
+            filled
+            hide-details
+            :value="business.name"
+          />
+          <div @click="business=null">
+            <v-icon color="primary">mdi-close</v-icon>
```

Comment:
This is a usable clear/retry button, but I'm still waiting for Janis' design on it.